### PR TITLE
DB-6923 pom file to build stand alone version (2.7)

### PR DESCRIPTION
**Pull request by perryluopl**
Sample command to build standalone package:

mvn clean package -Pcdh5.8.3 -DskipTests

The tar ball will be at spliceengine/assembly/standalone/target.

### CODE REVIEW

File: assembly/standalone/template/bin/start-splice.sh [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+## delete old logs before we start fresh
+/bin/rm -f *.log*
+
+ZOO_LOG="${RUN_DIR}/zoo.log"
+echo "Starting Zoo. Log file is ${ZOO_LOG}"
+
+
+# TODO eventually move - need cp script to consolidate as part of prepare package
+#CP="${BASE_DIR}/lib":"${BASE_DIR}/lib/*"
+CP="${BASE_DIR}/lib":"${BASE_DIR}/lib/*:${BASE_DIR}/conf"
```
Comment (MurrayBrown) [inline assembly/standalone/template/bin/start-splice.sh]: 👍

File: assembly/standalone/template/bin/start-splice.sh [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+pushd "${RUN_DIR}" > /dev/null
+## delete old logs before we start fresh
+/bin/rm -f *.log*
+
+ZOO_LOG="${RUN_DIR}/zoo.log"
+echo "Starting Zoo. Log file is ${ZOO_LOG}"
+
+
+# TODO eventually move - need cp script to consolidate as part of prepare package
+#CP="${BASE_DIR}/lib":"${BASE_DIR}/lib/*"
```
Comment (MurrayBrown) [inline assembly/standalone/template/bin/start-splice.sh]: why leave in a commented out line? the old line is still in source control?

File: hbase_sql/pom.xml [hunk 3]
```diff
@@ -774,6 +774,47 @@
             <plugin>
                 <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>############### clean jars ###############</id>
```
Comment (MurrayBrown) [inline hbase_sql/pom.xml]: is this the right name for this step? 

perhaps 'clean standalone jars' or just 'standalone build' ?

**Comment by perryluopl** [comment]
Jenkins Job to build standalone is at http://206.225.8.98:8080/job/spliceengine-2.7/job/spliceengine-standalone/

**Comment by wtang16** [comment]
@perryluopl On Jenkins, where is the build artifact stored?

**Pull request merged**